src/modelhub/inference_engines/rf3.py

```diff
@@ -474,7 +342,22 @@ def eval(self):
                 else out["asym_unit"][0]
             )
 
-            atom_array = self.prepare_atom_array(
+            # ... extract temlate information from the CIF file
+            templating_from_cif = category_to_dict(out["cif_block"], "templating")
+
+            # First, apply the template selection from the CIF file
+            atom_array = apply_conformer_and_template_selections(
```

Comment:
For the future: Consistent with the atomworks philosophy I would make these transforms in the inference transform pipeline that gets seeded with the information passed in via CLI. Not in scope for this PR.